### PR TITLE
Updated react-native-sdk

### PR DESCRIPTION
**Pull request by PurushYM**
- Updated android gradle library target to 1.8.+
- Exposed API
 - setStatusBarColour()
 - setCloseButtonColor()

### CODE REVIEW

File: ios/YMChatReactNative.m [hunk 1]
```diff
@@ -98,4 +108,13 @@ - (void) onBotClose {
     }
 }
 
+- (UIColor *)getColorFromHexString:(NSString *)hexString {
+    unsigned rgbValue = 0;
+    NSScanner *scanner = [NSScanner scannerWithString:hexString];
+    [scanner setScanLocation:1]; // bypass '#' character
+    [scanner scanHexInt:&rgbValue];
+    return [UIColor colorWithRed:((rgbValue & 0xFF0000) >> 16)/255.0 green:((rgbValue & 0xFF00) >> 8)/255.0 blue:(rgbValue & 0xFF)/255.0 alpha:1.0];
```
Comment (kaunteya) [inline ios/YMChatReactNative.m:116]: bypass '#' only if it exists in the first location

**Review by kaunteya — CHANGES_REQUESTED** [review]

File: ios/YMChatReactNative.m (hunk 1 above)
Comment (PurushYM) [reply ios/YMChatReactNative.m]: ok

File: ios/YMChatReactNative.m [hunk 2]
```diff
@@ -98,4 +109,17 @@ - (void) onBotClose {
...
+- (UIColor *)getColorFromHexString:(NSString *)hexString {
+    if ([(NSString*) hexString characterAtIndex:0] == '#')
+    {
+        unsigned rgbValue = 0;
+        NSScanner *scanner = [NSScanner scannerWithString:hexString];
+        [scanner setScanLocation:1]; // bypass '#' character
+        [scanner scanHexInt:&rgbValue];
+        return [UIColor colorWithRed:((rgbValue & 0xFF0000) >> 16)/255.0 green:((rgbValue & 0xFF00) >> 8)/255.0 blue:(rgbValue & 0xFF)/255.0 alpha:1.0];
+    }
+    return  nil;
```
Comment (kaunteya) [inline ios/YMChatReactNative.m:121]:
```
if charAtIndex == '#' {
 scanner setScanlocation: 1
} else {
 scanner setScanlocation: 0
}
```

**Review by kaunteya — CHANGES_REQUESTED** [review]

File: ios/YMChatReactNative.m (hunk 2 above)
Comment (PurushYM) [reply ios/YMChatReactNative.m:121]: ok

**Review by kaunteya — APPROVED** [review]

**Pull request merged**